### ISSUE TITLE
Add darwin arm64 LE build

### ISSUE THREAD

**Issue by mhart** [evanw/esbuild#550]
Currently get this on the latest Macs:

```
Unsupported platform: darwin arm64 LE
```

**Comment by mhart** [comment]
I realize this won't be possible until you've got access to Go that supports it: https://github.com/golang/go/issues/38485#issuecomment-730247329

But I figured it's worth having an issue to track it

**Comment by evanw** [comment]
I was wondering when this would come in. From what I understand, Go is currently in feature freeze for 1.15 and this isn't going to be supported by Go until the 1.16 release in February.

**Comment by evanw** [comment]
Sounds like esbuild should be working on the M1 soon. It turns out the simple way to fix this is just to ship the `darwin x64 LE` version. This was done in #564.

**Comment by RoXuS** [comment]
I got the following error:

![image](https://user-images.githubusercontent.com/3296671/101170982-09583480-35f4-11eb-8668-f85571716829.png)

This is the only one that does not work :(

If I can help?

**Comment by 285851937** [comment]
@evanw is there any way to install the esbuild on Raspberry Pi 4 ?
My Pi's info:
```
# uname -a
Linux raspberrypi 4.19.97-v7l+ #1294 SMP Thu Jan 30 13:21:14 GMT 2020 armv7l GNU/Linux
```
and after install esbuild,there is an error:
```
Unsupported platform: linux arm LE
```
ps: It's ``` arm``` NOT ```arm64```
Thanks~~

**Comment by evanw** [comment]
> Unsupported platform: linux arm LE

That's tracked by issue #528, not this issue (this issue is about macOS).

In theory your device should now be supported in version 0.8.22. I don't have a Raspberry Pi myself, so I have not tested it. Let me know in issue #528 if you have any problems.

**Comment by 285851937** [comment]
excuse me got a wrong issuse. I've been upgraded raspbian to 64bit version, like this:```Linux raspberrypi 5.4.79-v8+ #1373 SMP PREEMPT Mon Nov 23 13:32:41 GMT 2020 aarch64 GNU/Linux``` and install the **0.8.22** successfully. thank you a lot...
PS:[steps of upgrading **arm** to **arm64** for Raspberry Pi](https://www.mmuaa.com/post/0c9188ffde4e2cff.html) if any one need to upgrad  Raspberry Pi.

**Comment by sudarshansb143** [comment]
If someone is facing same issue with angular please refer here 

https://stackoverflow.com/questions/69613984/npm-install-esbuild-unsupported-warning

it worked for me  on angular `12.1.4`

**Comment by ajnart** [comment]
If this is solved, then how am I getting this error in late 2023 ? On my NextJS application running `yarn install` produces the following 
```log
➤ YN0007: │ esbuild@npm:0.5.26 must be built because it never has been before or the last one failed
➤ YN0009: │ esbuild@npm:0.5.26 couldn't be built successfully (exit code 1, logs can be found here: /private/var/folders/q8/mhk8syfs6tscrmprgglz4nhc0000gn/T/xfs-be161bca/build.log)
```

Can someone help me with that ? This issue is very frustrating

**Comment by lydell** [comment]
The logs you posted say `esbuild@npm:0.5.26` which makes it look like your are installing esbuild v0.5.26. This issue was fixed in [v0.8.47](https://github.com/evanw/esbuild/releases/tag/v0.8.47). The latest is version as of this writing is v0.19.5. Install a later version.
